doc/onboarding.md

```diff
@@ -100,18 +107,20 @@ onboarding session.
         (especially if it just has nits left).
   * Approving a change
     * Collaborators indicate that they have reviewed and approve of the
-        the changes in a pull request by commenting with `LGTM`, which stands
-        for "looks good to me".
-    * You have the power to `LGTM` another collaborator's (including TSC/CTC
-        members) work.
-    * You may not `LGTM` your own pull requests.
-    * You have the power to `LGTM` anyone else's pull requests.
+        the changes in a pull request using Github’s approval interface
+    * Some people like to comment `LGTM` (“Looks Good To Me”)
+    * You have the power to approve any other collaborator's work.
+    * You may not approve your own pull requests.
```

Comment:
You can't (GitHub does not allow it) so:
`You can't approve your own pull requests, so you'll need at least another Collaborator's approval for you own PRs.`